transactions/switchboard/add_vault_capability.cdc

```diff
@@ -0,0 +1,30 @@
+// This transaction is a template for a transaction that
+// could be used by anyone to add a new fungible token vault
+// capability to their switchboard resource
+
+import FungibleToken from "./../../contracts/FungibleToken.cdc"
+import FungibleTokenSwitchboard from "./../../contracts/FungibleTokenSwitchboard.cdc"
+import ExampleToken from "./../../contracts/ExampleToken.cdc"
+
+transaction {
+
+    let exampleTokenVaultCapabilty: Capability<&{FungibleToken.Receiver}>
+    let switchboardRef:  &FungibleTokenSwitchboard.Switchboard
+
+    prepare(signer: AuthAccount) {
+      // Get the example token vault capability from the signer's account
+      self.exampleTokenVaultCapabilty = 
+        signer.getCapability<&{FungibleToken.Receiver}>
+                                (ExampleToken.ReceiverPublicPath)
```

Comment:
I would recommend adding the check whether the capability exists because the failed transaction pollutes the blockchain and charges transaction costs. We can also save before submitting the transaction if we have a check in the transaction.  (If I am not wrong)